crates/bevy_ecs/src/component.rs

```diff
@@ -252,6 +293,15 @@ impl SparseSetIndex for ComponentId {
     }
 }
 
+/// A low-level descriptor for initializing [`Component`]s in a [`World`].
```

Comment:
I'd like to avoid reusing descriptor here: it doesn't tell the reader anything they didn't know.

Reply:
Reworded to use "configuration" instead.